package.json

```diff
@@ -27,22 +28,33 @@
     "precompile": "node -p \"'export const LIB_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
     "compile": "tsc -p ./",
     "lint": "yarn run prettier src -c",
-    "watch": "webpack --watch --devtool nosources-source-map --info-verbosity verbose --config ./build/extension.webpack.config.js",
+    "watch": "webpack --watch --devtool nosources-source-map --config ./build/extension.webpack.config.js",
     "watch2": "tsc -watch -p ./",
     "package": "vsce package",
     "publish": "vsce publish",
     "package-ext": "webpack --mode production --config ./build/extension.webpack.config.js",
     "pretest": "yarn run compile",
     "test": "node node_modules/mocha/bin/_mocha -u tdd --timeout 999999 --colors ./out/tests",
     "sbt": "sbt universal:packageBin",
-    "pre-build": "mv LICENSE tmp.LICENSE && mv NOTICE tmp.NOTICE && cp build/bin.LICENSE LICENSE && cp build/bin.NOTICE NOTICE",
+    "pre-build": "run-script-os",
+    "pre-build:default": "mv LICENSE tmp.LICENSE && mv NOTICE tmp.NOTICE && cp build/bin.LICENSE LICENSE && cp build/bin.NOTICE NOTICE",
+    "pre-build:windows": "move LICENSE tmp.LICENSE && move NOTICE tmp.NOTICE && copy build/bin.LICENSE LICENSE && copy build/bin.NOTICE NOTICE",
     "build": "yarn pre-build && yarn sbt && yarn install && yarn compile && yarn package && yarn post-build",
-    "post-build": "rm -rf NOTICE LICENSE && mv tmp.LICENSE LICENSE && mv tmp.NOTICE NOTICE"
+    "post-build": "run-script-os",
+    "post-build:default": "rm NOTICE && rm LICENSE && mv tmp.LICENSE LICENSE && mv tmp.NOTICE NOTICE",
+    "post-build:windows": "del /q NOTICE && del /q LICENSE && move tmp.LICENSE LICENSE && move tmp.NOTICE NOTICE"
   },
   "dependencies": {
+    "@grpc/grpc-js": "^1.5.4",
+    "@types/moo": "^0.5.5",
     "await-notify": "1.0.1",
     "child_process": "^1.0.2",
+    "google-protobuf": "^3.19.4",
     "hexy": "^0.3.1",
+    "minimist": "^1.2.5",
+    "moo": "^0.5.1",
+    "omega-edit": "0.9.7",
+    "run-script-os": "^1.1.6",
     "unzip-stream": "^0.3.1",
```

Comment:
I don't think minimist or moo are used so they can be removed. But I know omega-edit license is through Apache and will be bundled I believe aswell not sure on run-script-os but I believe run-script-os is a dev dependency only needed for the building process.

Reply:
Does package.json differentiate between build-only vs release dependencies. If it doesn't, and run-os and other dev only dependencies end up in the .vsix file somehow, then we still need to include the license.

Reply:
Yea so I mis-spoke I believe all of the are needed except minimist. But you are right there I believe any package listed in the package.json either dev or not will be added so I will get those added.

Reply:
@stevedlawrence Actually we don't list any of these dependencies in LICENSE, NOTICE, bin.LICENSE or bin.NOTICE because we don't copy the `node_modules` folder into the VSIX file so these wouldn't need to be added correct?

Reply:
Don't these dependencies get minimized and packaged into `extension/dist/ext/extension.js` in the .vsix file? They don't need to be in LICENSE or NOTICE, but if they end up in the .vsix file they need to be mentioned in bin.NOTICE or bin.LICENSE.

Reply:
I believe they do yes, it is very weird that file seems to call the packages but you don't have an actual `node_modules` folder where those packages installed. So I am not sure about adding all of the node packages into the bin.LICENSE or bin.LICENSE because even though we only require so many they requires there is a total of around 430 modules installed after you run `yarn install` that seems like a lot to try to add to these files